Cargo.toml

```diff
@@ -21,7 +21,7 @@ html5ever = "0.26"
 selectors = "0.25.0"
 smallvec = "1.11.0"
 tendril = "0.4.3"
-ahash = "0.8"
+ahash = { version = "0.8", default-features = false, features = ["std"] }
```

Comment:
I don't think we should disable the random seed unconditionally as it is an important security feature and we chose `ahash` specifically for its DoS resistance.

I suggest we do the following: On `cfg(target = "wasm32-unknown-unknown")`, we use `ahash` and its `const-random` feature whereas on `cfg(not(target = "wasm32-unknonw-unknown"))` we rely on `runtime-rng` as before.